README.md

```diff
@@ -135,11 +135,22 @@ The other placeholders are specified separately.
 See [examples](examples/README.md) for configuration examples.
 
 ```yml
-# Client ID of managed identity
+# Azure credentials choice section. Only one of the following flags should be set ot true to indicate which credentials to use. Otherwise an exception would be raised by the validator. 
```

Comment:
```suggestion
# Azure credentials choice section. Only one of the following flags should be set to true to indicate which credentials to use. Otherwise an exception would be raised by the validator. 
```